Samples/TardisBank/Solutions/Suteki.TardisBank.Web.Mvc/Global.asax.cs

```diff
@@ -75,6 +62,18 @@ protected void Application_Start()
             BundleConfig.RegisterBundles(BundleTable.Bundles);
         }
 
+        ISessionFactory CreateSessionFactory(IKernel kernel)
```

Comment:
@shatl would be better to move the NHibernate registration stuff into it's own installer